src/compiler/objective_c_plugin.cc

```diff
@@ -90,8 +90,10 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
       return true;
     }
 
+    bool grpc_local_import;
```

Comment:
do we need to default this to false?

Reply:
Yes, we need to default this to false, this is only used for first party users.